azurerm/internal/services/apimanagement/api_management_backend_resource_test.go

```diff
@@ -52,10 +52,10 @@ func TestAccApiManagementBackend_allProperties(t *testing.T) {
 				check.That(data.ResourceName).Key("credentials.0.authorization.0.parameter").HasValue("parameter"),
 				check.That(data.ResourceName).Key("credentials.0.authorization.0.scheme").HasValue("scheme"),
 				check.That(data.ResourceName).Key("credentials.0.certificate.0").Exists(),
-				check.That(data.ResourceName).Key("credentials.0.header.header1").HasValue("header1value1"),
-				check.That(data.ResourceName).Key("credentials.0.header.header2").HasValue("header2value1"),
-				check.That(data.ResourceName).Key("credentials.0.query.query1").HasValue("query1value1"),
-				check.That(data.ResourceName).Key("credentials.0.query.query2").HasValue("query2value1"),
+				check.That(data.ResourceName).Key("credentials.0.header.header1").HasValue("header1,value1"),
+				check.That(data.ResourceName).Key("credentials.0.header.header2").HasValue("header2,value1"),
+				check.That(data.ResourceName).Key("credentials.0.query.query1").HasValue("query1,value1"),
+				check.That(data.ResourceName).Key("credentials.0.query.query2").HasValue("query2,value1"),
```

Comment:
Looks like the config has different values here?
```
     header = {
      header1 = "header1value1,header1value2"
      header2 = "header2value1,header2value2"
    }
    query = {
      query1 = "query1value1,query1value2"
      query2 = "query2value1,query2value2"
    }
```

Reply:
Ooo, good call. Got it